neo/Prompt/Commands/Show.py

```diff
@@ -0,0 +1,159 @@
+import os
+import psutil
+import datetime
+from neo.Prompt.CommandBase import CommandBase, CommandDesc, ParameterDesc
+from neo.Prompt.PromptData import PromptData
+from neo.Prompt.Utils import get_arg
+from neo.Core.Blockchain import Blockchain
+from neocore.UInt256 import UInt256
+from neo.IO.MemoryStream import StreamManager
+from neo.Network.NodeLeader import NodeLeader
+from neo.Implementations.Notifications.LevelDB.NotificationDB import NotificationDB
+from neo.logging import log_manager
+import json
+
+
+logger = log_manager.getLogger()
+
+
+class CommandShow(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+        self.register_sub_command(CommandShowBlock())
+        self.register_sub_command(CommandShowHeader())
+        self.register_sub_command(CommandShowTx())
+        self.register_sub_command(CommandShowMem())
+        self.register_sub_command(CommandShowNodes(), ['node'])
+
+    def command_desc(self):
+        return CommandDesc('show', 'show data from the blockchain')
```

Comment:
I think we need to generalise the description a bit more because it also includes "node memory usage", "connected nodes", "overall client state" (and who knows what in the future).

I'm out of inspiration at this point, but I'm sure you can find a nice and short generalised line 👍

Reply:
How about "show useful data"
?